app-misc/rpick/rpick-0.4.0.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 2017-2019 Gentoo Authors
```

Comment:
2017 feels pretty random in here? :) Just leave 2019.

Reply:
Yeah - I used cargo-ebuild to generate this ebuild, as recommended by the [Gentoo Rust Project](https://wiki.gentoo.org/wiki/Project:Rust), and it set these years. It also set the ```RESTRICT="mirror"``` (and I am not sure about it, as I don't know what it does, but I will go learn now ☺).

I'll change this to 2019.